services/minespace-web/src/styles/overrides/filepond-overrides.scss

```diff
@@ -75,8 +65,8 @@
 
 /* bordered drop area */
 .filepond--panel-root {
-  background-color: transparent;
-  border: 2px solid $primary-color;
+  background-color: rgba(230, 235, 240, 1); /* the background color of the filepond drop area */
```

Comment:
Instead of a comment, would prefer to assign to variable. $filepond-drop-bg or something. 

I have a preference for hex format when we don't need the alpha channel, but that's probably just me. 😁

Reply:
Updated the code accordingly.